CredScanSuppressions.json

```diff
@@ -0,0 +1,9 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+      {
+        "placeholder": "  hasWebHookSecret: boolean",
+        "_justification": "The word secret is not a secret"
+      }
```

Comment:
Deep insight.

Reply:
If only we could teach that to the credscan ML